lib/spack/spack/test/compilers/conversion.py

```diff
@@ -6,9 +6,11 @@
 
 from spack.compilers.config import CompilerFactory
 
+pytestmark = [pytest.mark.usefixtures("mock_compilers")]
```

Comment:
Why do we need this, instead of _not_ defining a new fixture and:
```suggestion
pytestmark = [pytest.mark.usefixtures("config", "mock_packages")]
```
?

Reply:
That was residual from an abandoned investigation. Thanks for catching it!